src/design-system/whats-changed.md.njk

```diff
@@ -13,6 +13,245 @@ layout: layout-single-page-prose.njk
 
 The tables below show the old and new names for components, classes and mixins, to help you find what you need.
 
+## Template
+
+Previously to keep your pages consistent with the rest of GOV.UK you would have used [GOV.UK Template](http://alphagov.github.io/govuk_template/).
+
+We have brought this into GOV.UK Frontend and it can be consumed via Nunjucks or HTML.
+
+If you are not using Nunjucks you can get the HTML from the [Template guidance page](../styles/template/).
+
+### Nunjucks
+
+Where possible we have kept the Template API the same, with the main changes being camelCase vs snake_case.
+
+<table class="govuk-table app-table--fixed">
+  <caption class="govuk-table__caption govuk-visually-hidden">Nunjucks Template</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header" scope="col">GOV.UK Template</th>
+      <th class="govuk-table__header" scope="col">GOV.UK Frontend</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">:top_of_page</td>
+      <td class="govuk-table__cell">Deprecated: Putting content above the <code>&lt;!DOCTYPE html&gt;</code> will result in broken pages for users of older Internet Explorer versions.</td>
```

Comment:
change 'Deprecated:' to 'deprecated:' to follow style of all other instances on this page

Reply:
They all seem to be capitalised, lemme know if I've missed something.